README.md

```diff
@@ -510,11 +510,11 @@ Underreact always runs [Autoprefixer](https://github.com/postcss/autoprefixer) f
 
 ### publicAssetsPath
 
-Type: `string`. Default: `underreact-assets`.
+Type: `string`. Default: `assets`.
```

Comment:
I actually think that `underreact-assets` might be better equipped to facilitate long-term caching. It might be risky to set long-term caching on anything in in an `/assets/` path, because that's such a generic word; but with `/underreact-assets/`, we can be confident that those files were generated by Underreact.

I understand that this means Batfish should use `batfish-assets` or something like that — I think that would be a good change.

Reply:
My online concern would be to keep a track of different names with different build tools. Can you think of a non-generic word which we can use for caching.

Reply:
`hashed-assets`?

Reply:
I'm not too concerned about different names for different tools, because I think people should be very careful and deliberate about setting cache-control headers — not expect a convention work magic for them.